blockly-rtc/src/http_handlers.js

```diff
@@ -20,10 +20,18 @@
  * @author navil@google.com (Navil Perez)
  */
 
+/**
+ * A local representation of an entry in the database.
+ * @typedef {Object} LocalEntry
+ * @property {<!Array.<!Object>>} events An array of Blockly Events in JSON
+ * format.
+ * @property {string} entryId The id assigned to an event by the client.
+ */
+
 /**
  * Query the database for rows since the given server id.
  * @param {number} serverId serverId for the lower bound of the query.
- * @return {!Array.<!Object>} Rows of events since the given serverId.
+ * @return {<!Array.<!Object>>} Rows of entries since the given serverId.
```

Comment:
Can you make `!Object` more specific here?  Is it `!LocalEntry`?

Reply:
They are a bit different. LocalEntry is how they are stored locally before being written to the database where they then assigned a serverId.
LocalEntry
```
{
  events:
  entryId:
}
```

This object (tentatively ServerEntry)
```
{
  events:
  entryId:
  serverId:
}
```
 I can make another typedef for this object and make it more specific.

Reply:
Actually, there already is a typedef for this called "Row" I will change the JSDoc accordingly.